docker/Dockerfile.centos-8

```diff
@@ -2,7 +2,7 @@ FROM quay.io/centos/centos:stream8
 
 # Add runtime dependencies.
 RUN dnf install -y \
-        http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm \
+        http://resources.ovirt.org/pub/yum-repo/ovirt-release44.rpm \
```

Comment:
This does not look right.

Reply:
release-master was not available, and this was the most recent rpm I saw in the same resources folder. Maybe we need to upload new rpms? Not sure.

Reply:
The issue is old links in the docker files. The right way to install is explained
in ovirt master snapshot copr repo. You need to update all the docker files to consume
the right repos.
https://copr.fedorainfracloud.org/coprs/ovirt/ovirt-master-snapshot/

Reply:
Ah I see, I should've known.

I'll update them :+1: